Source/SwiftLintFramework/Rules/Performance/EmptyCountRule.swift

```diff
@@ -29,12 +29,16 @@ public struct EmptyCountRule: ConfigurationProviderRule, OptInRule, AutomaticTes
             "[Int]().↓count == 0x00_00\n",
             "[Int]().↓count == 0b00\n",
             "[Int]().↓count == 0o00\n",
-            "↓count == 0\n"
+            "↓count == 0\n",
+            "[Int]().↓count < 1\n",
+            "[Int]().↓count < 0x001\n",
+            "[Int]().↓count < 0b1\n",
+            "[Int]().↓count < 0o01\n"
         ]
     )
 
     public func validate(file: File) -> [StyleViolation] {
-        let pattern = "\\bcount\\s*(==|!=|<|<=|>|>=)\\s*0(\\b|([box][0_]+\\b){1})"
+        let pattern = "\\bcount\\s*(((==|!=|<|<=|>|>=)\\s*0(\\b|([box][0_]+\\b){1}))|<\\s*(1|0[box]0*1){1}\\b)"
```

Comment:
Would you consider change the regex to `"\\bcount\\s*(((==|!=|<|<=|>|>=)\\s*0(\\b|([box][0_]+\\b){1}))|<\\s*(1|0[box]0*1){1}\\b|>=\\s*1(\\b|([box][0_]+\\b){1}))"`?

That would also cover `count >= 1`

Reply:
It changes `...|<\\s*...` to `...|(<|>=)\\s*...`

Reply:
<img width="399" alt="Screen Shot 2019-12-06 at 3 44 28 pm" src="https://user-images.githubusercontent.com/308592/70296813-b8c2c580-183f-11ea-9ca3-2588094a6545.png">